modules/incentives/src/lib.rs

```diff
@@ -317,10 +317,25 @@ pub mod module {
 					(pending_reward.saturating_sub(deduction_amount), deduction_amount)
 				};
 
-				// transfer the actual reward(pending reward exclude deduction) to user from the pool. it should not
-				// affect the process, ignore the result to continue. if it fails, just the user will not
-				// be rewarded, there will not increase user balance.
-				T::Currency::transfer(currency_id, &Self::account_id(), &who, actual_amount)?;
+				let res = T::Currency::transfer(currency_id, &Self::account_id(), &who, actual_amount);
+				if let Err(e) = res {
+					log::warn!(
+						target: "incentives",
+						"transfer: failed to transfer {:?} {:?} from {:?} to {:?}: {:?}. \
+						This is unexpected but should be safe",
+						actual_amount, currency_id, Self::account_id(), who.clone(), e
```

Comment:
```suggestion
						actual_amount, currency_id, Self::account_id(), &who, e
```

Reply:
this is not really unexpected that now we know what could trigger this, so better to add a comment to explain